packages/runtime-common/index-query-engine.ts

```diff
@@ -462,6 +464,19 @@ export class IndexQueryEngine {
     return { prerenderedCards, scopedCssUrls: [...scopedCssUrls], meta };
   }
 
+  async fetchCardTypeSummary(realmURL: URL): Promise<CardTypeSummary[]> {
+    let results = (await this.query([
+      `SELECT value
+       FROM realm_meta r
+       INNER JOIN realm_versions rv ON r.realm_url = rv.realm_url
+       WHERE`,
+      ...every([['r.realm_url =', param(realmURL.href)]]),
```

Comment:
you need to also include only realm_meta rows that are equal to the `rv.current` version